AdRulesIMAWidevineSampleApp/src/main/java/com/brightcove/player/samples/imawidevine/adrules/MainActivity.java

```diff
@@ -60,7 +60,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Create the catalog object which will start and play the video.
         Catalog catalog = new Catalog("FqicLlYykdimMML7pj65Gi8IHl8EVReWMJh6rLDcTjTMqdb5ay_xFA..");
```

Comment:
How does one translate a video of choice into one of these opaque strings?  Is there not a way to provide some sort of id and a string name/id for a video one desires?  Would this be the Wedge API?